bin/limbo

```diff
@@ -11,7 +11,15 @@ parser.add_argument('-c', dest="command", help='run a single command')
 parser.add_argument('--database', '-d', dest='database_name', default='limbo.sqlite3',
                     help="Where to store the limbo sqlite database. Defaults to limbo.sqlite3")
 parser.add_argument('--pluginpath', '-pp', dest='pluginpath', default=None,
-                    help="The path where limbo should look to find its plugins")
+                    help=("The primary path where limbo should look to find its plugins. Setting this "
+                          "will override the default list of plugins."))
+parser.add_argument('--extrapluginpaths', '-epp', dest='extrapluginpaths', default=[],
```

Comment:
Could we just modify `--pluginpath` to accept multiple instances instead of adding a new flag?

Reply:
I put some suggestions in a branch `llimllib-suggestions`. Here's the diff: https://github.com/appleshampooid/limbo/compare/appleshampooid_additive_plugins...llimllib:limbo:llimllib-suggestions

How do they look to you?

Reply:
> Could we just modify `--pluginpath` to accept multiple instances instead of adding a new flag?

Yes, I like that idea better. My only concern is still having the ability to load all the default plugins, and extras. I think it can be done though, stand by.

Reply:
> Could we just modify `--pluginpath` to accept multiple instances instead of adding a new flag?

I generally like this idea better, other than the somewhat-clunky nature of including the default plugins along with your custom ones (see my updated PR description for an example).

Let me know what you think? Could also include the default plugins by default, all the time, and then maybe an extra param to remove them? Shrug.